lib/seattleflu/id3c/cli/command/etl/clinical.py

```diff
@@ -275,6 +277,13 @@ def insurance(insurance_response: Optional[Any]) -> list:
         "medicare": "government",
         "tricare": "government",
         "other": "other",
+        "care": "other",
+        "caid": "other",
+        "comm": "other",
+        "self": "other",
+        "tce": "other",
+        "case rate": "other",
+        "wc": "other",
```

Comment:
Is the idea here that this lets the ETL complete on all existing data but in the future we'll update these new mappings to non-other values and bump the revision again?

Reply:
Izzy thinks (and I agree) safe to assume that CARE is Medicare (government), CAID is Medicaid (government), COMM is Commercial (private). Can we map those in a new commit for the benefit of the UW data?

Reply:
> Izzy thinks (and I agree) safe to assume that CARE is Medicare (government), CAID is Medicaid (government), COMM is Commercial (private). Can we map those in a new commit for the benefit of the UW data?

Yes, absolutely

Reply:
> Is the idea here that this lets the ETL complete on all existing data but in the future we'll update these new mappings to non-other values and bump the revision again?

Yes and no. The ETL does complete on all existing data, but in the future, if an unknown value is given, it will break.
I don't think the revision number would need to be bumped again if we're adding more values that code to "other" or "None". However if we are to change existing mapped values, then yes we would need to bump the revision number.